modules/Fetch.js

```diff
@@ -141,9 +143,11 @@ class Fetch extends Component<Props> {
       path,
       url,
       timeout,
+      onIntercept,
     } = props
     let route: ?string
     let timeoutValue: number = 0
+    let interceptor: ?Interceptor
```

Comment:
I don't know if GitHub lost my review on this line but you don't need this variable actually